### ISSUE TITLE
The West Live advert - how to permanently suppress

### ISSUE THREAD

**Issue by Dibrom2** [uBlockOrigin/uAssets#16688]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Website: `https://thewest.com.au`

Example article: 
`https://thewest.com.au/news/wa/perth-bushfires-yangebup-man-charged-over-a-series-of-fires-across-perths-southern-suburbs-c-9713572`

In the middle of the story there is a large inserted advert for a radio show which is in a frame of some sort. I can select this frame using the UBO eyedropper and it is blocked, but it will only be blocked on this one page and this one story. Go to another page or another story and it will be back again, so it's a perpetual game of whack-a-mole. How can I get UBO to target this frame with the radio show advert no matter what page or story it appears on?

I think the problem is that the frame contents change for every story because it contains the headlines of the news stories of that day. How can I get around that in the filtering please?

### URL(s) where the issue occurs.

```
https://thewest.com.au/news/wa/perth-bushfires-yangebup-man-charged-over-a-series-of-fires-across-perths-southern-suburbs-c-9713572

https://thewest.com.au/politics/state-politics/retiring-alannah-mactiernan-laments-infantilisation-of-indigenous-aussies-backs-hardline-stance-on-parents-c-9714335

https://thewest.com.au/news/albany-advertiser/washed-up-beach-package-adds-to-afp-investigation-after-suspicious-packages-found-on-capsized-boat--c-9715688
```

### Screenshot(s)

https://user-images.githubusercontent.com/91712917/218041002-e590acfe-2718-4cc7-9df9-74d38be3ec0e.png

### uBO version

1.46.0

### Browser name and version

Firefox x64 107.0.1

### Settings

None

### Notes

Note that the website in question is subscription only and has a very aggressive paywall. Will need Bypass Paywalls Clean extension installed to access to replicate the problem

https://gitlab.com/magnolia1234/bypass-paywalls-firefox-clean

**Comment by krystian3w** [comment]
> how to permanently suppress

No possible.

All they have to do is activate notifications with commits for filtering what line was modified and update the code on the server.

Or the server will exhibit polymorphic code at intervals beyond the capacity of eager volunteers..

**Comment by krystian3w** [comment]
```
||omny.fm/shows/the-west-live/$frame,object,domain=thewest.com.au
! or
thewest.com.au##embed[src*="omny.fm/shows/the-west-live"], iframe[src*="omny.fm/shows/the-west-live"]
```

But can down if page will fight with 1 user from 8 billions.

**Comment by Dibrom2** [comment]
Ahhh, OK. "No Possible" means Not Possible ? Basically you're saying "too hard, not worth the bother"?

**Comment by krystian3w** [comment]
This could turn into a battle with windmills.

https://en.wikipedia.org/wiki/Tilting_at_windmills_(disambiguation)

**Comment by mapx-** [comment]
enough adding

`||omny.fm^$object,domain=thewest.com.au`

it seems to be some sort of self-promotion, just use as custom filter
